src/js/stores/DCOSStore.js

```diff
@@ -146,6 +148,12 @@ class DCOSStore extends EventEmitter {
 
   }
 
+  get jobTree() {
```

Comment:
I think the Jobs belong here as much as the Services do. Or am I missing the point of having the DCOSStore?